docs/apereo-incubation.md

```diff
@@ -288,6 +288,8 @@ Next actions:
 
 Long range plans don't so much exist.
 
+cf. [roadmap.md][]
```

Comment:
docs\apereo-incubation.md
530  291:5-291:19  warning  Found reference to undefined definition  no-undefined-references  remark-lint

Reply:
Interesting. For some reason I thought that shorthand would work, but that's not a thing.

Reply:
For that a literal link syntax may work `<roadmap.md>`